UnitsNet/GeneratedCode/Quantities/ReciprocalLength.g.cs

```diff
@@ -630,11 +638,10 @@ public static bool TryParseUnit(string str, IFormatProvider? provider, out Recip
             return ReciprocalArea.FromInverseSquareMeters(left.InverseMeters * right.InverseMeters);
         }
 
-        /// <summary>Calculates the inverse of this quantity.</summary>
-        /// <returns>The corresponding inverse quantity, <see cref="Length"/>.</returns>
-        public Length Inverse()
```

Comment:
Maybe this was just this one file, you can disregard this.

Reply:
Not just this one, this happens in `Area`, `ReciprocalArea`, `Length` and `ReciprocalLength`.

But I think it's an improvement, `Inverse` is now always the first operator in the #region because it's now defined as
```
1 = Area.SquareMeter * ReciprocalArea.InverseSquareMeter
```
instead of
```
ReciprocalArea.InverseSquareMeter = 1 / Area.SquareMeter
```
and sorted on that string alphanumerically.